src/functions/schema.ts

```diff
@@ -14,6 +14,10 @@ export interface ISchemaOptions {
   schema: object;
   // The oasVersion, either 2 or 3 for OpenAPI Spec versions, could also be 3.1 or a larger number if there's a need for it, otherwise JSON Schema
   oasVersion?: Optional<number>;
+  allErrors?: boolean;
+
+  // this is used by oasDocumentSchema function, to be removed once we sort out
```

Comment:
@P0lip What's the foreseen plan here with regards to `schema` vs `oasDocumentSchema`? Should the later eventually replace the former? Do you foresee to keep them both?

Reply:
> @P0lip What's the foreseen plan here with regards to schema vs oasDocumentSchema? Should the later eventually replace the former? Do you foresee to keep them both?

I'm eyeing replacing ajv with some other schema validator, that, hopefully, produces more human-friendly errors.
For the time being, we need to prepare the results on our own.
If you need some dedicated treatment in case of errors reported by asyncapi-schema rule (assuming we have such), feel free to spin `asyncDocumentSchema` fn.

Reply:
Thx!